### PR TITLE
[PERF-496]Create JMeter Script for View Bibs (Titles), Holdings

### PR DESCRIPTION
**Pull request by KyryloShulzhenko**
## Links
https://issues.folio.org/browse/PERF-496
## Purpose
add JMeter scripts for view bibs and holdings

### CODE REVIEW

File: workflows-scripts/inventory/view-bibs/jmeter-supported-data/keywords.csv [hunk 1]
```diff
@@ -0,0 +1,1356 @@
+"A. A",2
```
Comment (mtraneis) [inline workflows-scripts/inventory/view-bibs/jmeter-supported-data/keywords.csv:1]: How did this file generate the loop count? Is it random?

Comment (mtraneis) [reply workflows-scripts/inventory/view-bibs/jmeter-supported-data/keywords.csv:1]: Will it loop over this file once it reaches the end of the file if the duration of the test runs longer than this list can supply the words?

File: workflows-scripts/inventory/view-holdings/jmeter-supported-data/keywords.csv
Comment (mtraneis) [inline workflows-scripts/inventory/view-holdings/jmeter-supported-data/keywords.csv:1]: Same questions for this file as in the other keywords.csv

**Review by mtraneis — APPROVED** [review]

File: workflows-scripts/inventory/view-bibs/jmeter-supported-data/keywords.csv (hunk 1 above)
Comment (KyryloShulzhenko) [reply workflows-scripts/inventory/view-bibs/jmeter-supported-data/keywords.csv:1]: Hello @mtraneis, It is not random it SQL script select substring(title, 1,4), count(1) -2 from holdings_record group by substring(title, 1,4) having count(1) >3 and count(1)<7.

File: workflows-scripts/inventory/view-bibs/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- mod-source-record-storage-5.5.2
+- okapi-4.14.7
+##### Frontend:
+- folio_inventory-9.2.8
+
+- FOLIO release: Nolana
+###### Csv data config files
+They are used for test parameterization, for creating different data sets for different users in the same test script.
+- credentials.csv (contains username,password,tenant, they are needed for login)
+- keywords.csv (contains keyword) in this case it is Instance keyword
```
[the remark this answers is not in the document]
Comment (KyryloShulzhenko) [reply workflows-scripts/inventory/view-bibs/README.md]: done

File: workflows-scripts/inventory/view-bibs/inventory_viewBibs.jmx [hunk 3]
```diff
@@ -0,0 +1,1661 @@
...
+                    </elementProp>
+                  </collectionProp>
+                </HeaderManager>
+                <hashTree/>
+              </hashTree>
+            </hashTree>
+          </hashTree>
+        </hashTree>
+      </hashTree>
+      <JSR223Listener guiclass="TestBeanGUI" testclass="JSR223Listener" testname="Log Failed Samples" enabled="false">
```
[the remark this answers is not in the document]
Comment (KyryloShulzhenko) [reply workflows-scripts/inventory/view-bibs/inventory_viewBibs.jmx]: I`ve removed

File: workflows-scripts/inventory/view-bibs/inventory_viewBibs.jmx [hunk 4]
```diff
@@ -0,0 +1,1661 @@
...
+	File failed_samples = new File(vars.get(&quot;global_BaseDir&quot;) + &quot;/failed_samples_${__machineIP()}.txt&quot;);
+	failed_samples.append(toLog);
+
+	} 
+
+</stringProp>
+        <stringProp name="scriptLanguage">groovy</stringProp>
+      </JSR223Listener>
+      <hashTree/>
+      <ResultCollector guiclass="ViewResultsFullVisualizer" testclass="ResultCollector" testname="View Results Tree" enabled="true">
```
[the remark this answers is not in the document]
Comment (KyryloShulzhenko) [reply workflows-scripts/inventory/view-bibs/inventory_viewBibs.jmx]: I disabled Veiew Results Tree

File: workflows-scripts/inventory/view-bibs/inventory_viewBibs.jmx [hunk 5]
```diff
@@ -0,0 +1,1661 @@
...
+      <Arguments guiclass="ArgumentsPanel" testclass="Arguments" testname="Environment &amp; Configuration" enabled="true">
+        <collectionProp name="Arguments.arguments">
+          <elementProp name="global_BaseDir" elementType="Argument">
+            <stringProp name="Argument.name">global_BaseDir</stringProp>
+            <stringProp name="Argument.value">${__BeanShell(import org.apache.jmeter.services.FileServer; FileServer.getFileServer().getBaseDir();)}${__BeanShell(File.separator,)}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="HOSTNAME" elementType="Argument">
+            <stringProp name="Argument.name">HOSTNAME</stringProp>
+            <stringProp name="Argument.value">${__P(HOSTNAME, okapi-ptf-nolana-2.int.aws.folio.org)}</stringProp>
```
[the remark this answers is not in the document]
Comment (KyryloShulzhenko) [reply workflows-scripts/inventory/view-bibs/inventory_viewBibs.jmx]: It was replaced on {HOSTNAME}

**Pull request merged**